client/src/app/shared/models/event-management/meeting.ts

```diff
@@ -20,10 +22,14 @@ export type AgendaItemCreation = 'always' | 'never' | 'default_yes' | 'default_n
  */
 export type BallotPaperSelection = 'NUMBER_OF_DELEGATES' | 'NUMBER_OF_ALL_PARTICIPANTS' | 'CUSTOM_NUMBER';
 
-export interface Settings {
+const MEETING_NAME = 'meeting';
+export class Settings {
     // Old "general_*" configs
+    @DataClassProperty([], MEETING_NAME)
     name: string;
+    @DataClassProperty([], MEETING_NAME)
     description: string;
+    @DataClassProperty([], MEETING_NAME)
     location: string;
```

Comment:
As discussed I have thought about another way to achieve those information we need at runtime.
Here is a more "spring-boot-like" approach:
The properties, which can be sent in a specific action to the server, are decorated with such a decorator. To distinguish between different actions and different payloads there are different "namespaces" we define by a string (our actual actions), like `meeting.create` or `meeting.update`.